Makefile

```diff
@@ -49,9 +49,12 @@ build: ## run tox / run tests and lint
 	tox
 
 docs: ## generate Sphinx HTML documentation, including API docs
-	rm -f docs/aicsimageio.rst
+	rm -f docs/aicsimageio*.rst
```

Comment:
Sorry if this sounds weird but why is there a Makefile in this project at all?

Reply:
I use it to run tests, generate docs, clean up from builds, etc.